src/typeahead/typeahead.ts

```diff
@@ -397,6 +397,18 @@ export class NgbTypeahead implements ControlValueAccessor, OnInit, OnChanges, On
 		this._nativeElement.value = toString(value);
 	}
 
+	private getAnnounceLocalizedMessage(count: number): string {
```

Comment:
Maybe it would be better to introduce an `@Input()` for these kind of translations as well?


Like `@Input() liveAnnouncerMessage: (count: number) => string;`

With default being the function you're suggesting I guess.

Reply:
Also in this project private methods start with `_` → something like `_getLiveAnnouncerLocalizedMessage`

Reply:
Regarding the `@Input`, I'm not really sure tbh, I'm not super familiar with the way `@angular/localize` works but from what I understood, people using this library will generate/maintain translation files based on what is flagged with `i18n`/`$localize`. Do we really want/need to provide an additional way to override the translations on top of this already existing mechanism ? Provided keys for this use case seem to cover every possibilities, WDYT ?